internal/vault/vault.go

```diff
@@ -33,16 +34,41 @@ func UnmarshalPKIIssueResponse(resp *api.Secret) (*PKICertResponse, error) {
 	return result, nil
 }
 
+// MarshalSecretData returns Kubernetes Secret data from an api.Secret
+// TODO: move to internal/helpers
```

Comment:
Not sure I follow why this should be moved to `internal/helpers/`. It looks specific to vault?

Reply:
Fixed in 7fa55d5ffeda41e8c2705cc32de3385a04e0ce1e

Reply:
Ah, it's now vault-agnostic, nice!